x/model/types/errors.go

```diff
@@ -131,3 +132,8 @@ func NewErrModelVersionDeletionCertified(vid interface{}, pid interface{}, softw
 		"Model version associated with vid=%v, pid=%v and softwareVersion=%v has a compliance record and can't be deleted",
 		vid, pid, softwareVersion)
 }
+
+func NewErrOtaChecksumIsNotValid(otaChecksum string) error {
+	return errors.Wrapf(ErrOtaChecksumIsNotValid,
+		"OtaChecksum %v is not base64 encoded", otaChecksum)
```

Comment:
I think the name of the function and the error message do not match, we need to correct the message or the name of the function